### DISCUSSION TITLE
integrate into homekit as a scene

### DISCUSSION THREAD

**Discussion by JensJensenn** [hugobloem/stateful_scenes#26] [Ideas]
The integration into homekit makes the stateful scene in HA a switch in Homekit. It would be nice, if the Ha stateful scene showed up in Homekit as a scene also.

**Comment by hugobloem** [comment]
Sounds like a good idea and worth looking into!

**Reply by JensjJ8200** [reply]
Thank you. I have the home+ app for ios, where it can be done.

**Comment by sayam93** [comment]
I think this has to be implemented in Home Assistant using Homekit Accessory Protocol. All accessories are exposed to Home app using the Homekit Bridge integration which uses this protocol. Philips Hue uses the same to export scenes to Home app on iOS. I don't think it can be done without tweaking the Homekit Bridge integration so that it allows the selected entities to be exposed as scenes.

**Reply by JensjJ8200** [reply]
Thank you for looking into it :-)
